conf/templates/default/local.conf.sample

```diff
@@ -242,9 +240,7 @@ PACKAGECONFIG:append:pn-nativesdk-qemu = " sdl"
 # this doesn't mean anything to you.
 CONF_VERSION = "2"
 
-BBMASK += "poky/meta/recipes-devtools/rust"
-BBMASK += "poky/meta/recipes-devtools/cargo"
 PREFERRED_VERSION_rust ?= "1.65%"
```

Comment:
is it needed still ?

Reply:
Scarthgap doesn't support the external meta-rust. You must use the OpenEmbedded version now, which is built-in. BBMASK prevents those built-in recipes from being used.